packages/install-expo-modules/src/plugins/cli/withCliIntegration.ts

```diff
@@ -65,10 +65,28 @@ const withCliBabelConfig: ConfigPlugin = (config) => {
   return withDangerousMod(config, [
     'ios',
     async (config) => {
-      const babelConfigPath = path.join(config.modRequest.projectRoot, 'babel.config.js');
-      let contents = await fs.promises.readFile(babelConfigPath, 'utf8');
-      contents = updateBabelConfig(contents);
-      await fs.promises.writeFile(babelConfigPath, contents);
+      const babelConfigJsPath = path.join(config.modRequest.projectRoot, 'babel.config.js');
+
+      if (await fs.promises.exists(babelConfigJsPath)) {
+        let contents = await fs.promises.readFile(babelConfigJsPath, 'utf8');
+        contents = updateBabelConfig(contents);
+        await fs.promises.writeFile(babelConfigJsPath, contents);
+        return config;
+      }
+      
+      const babelConfigJsonPath = path.join(config.modRequest.projectRoot, 'babel.config.json');
+
+      if (await fs.promises.exists(babelConfigJsonPath)) {
+        let contents = await fs.promises.readFile(babelConfigJsonPath, 'utf8');
+        contents = updateBabelConfig(contents);
```

Comment:
I'm afraid the result might be invalid json syntax: https://github.com/expo/expo/blob/2de789861daaeb6e658c4d5b2c12415ad61b8b6d/packages/install-expo-modules/src/plugins/cli/withCliIntegration.ts#L116

Reply:
Nice catch. It wasn't tested - expo repository was too big to even clone, so I had to write it from GitHub.com.

Reply:
I think this should be fine now!